### PR TITLE
Assignment Due Date

### PR DESCRIPTION
**Pull request by steven-solar**
This PR allows the club admin to set the due date of assignments posted, using the materialUI `TextField`.

### CODE REVIEW

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx [hunk 1]
```diff
@@ -81,10 +81,12 @@ class ClubPage extends Component {
       alert('Assignment creation failed. You do not own this club.');
       return;
     }
+    const dueDate = document.getElementById("due-date").value;
```
Comment (MauroGuerrero20) [inline capstone-frontend/src/components/ClubPage.jsx:84]: nit: use single quotations for string literals

File: capstone-frontend/src/components/ClubPage.jsx [hunk 2]
```diff
@@ -153,7 +155,18 @@ class ClubPage extends Component {
               <Form.Label> Post a new assignment! </Form.Label>
               <Form.Control as='textarea' rows='3' placeholder='Enter assignment text...' />
             </Form.Group>
-            <Button variant='primary' type='submit'> Submit </Button>
+            <div>
+              <TextField
+                id="due-date"
+                label="Due Date"
+                type="datetime-local"
```
Comment (MauroGuerrero20) [inline capstone-frontend/src/components/ClubPage.jsx:162]: nit: use single quotations for string literals

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (MauroGuerrero20) [inline capstone-frontend/src/components/ClubPage.jsx]: nit: use single quotations for string literals

File: capstone-frontend/src/components/ClubPage.jsx (hunk 2 above)
Comment (MauroGuerrero20) [inline capstone-frontend/src/components/ClubPage.jsx]: nit: use single quotations for string literals

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

Forgot to add my comments.

**Review by VivekShankar — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx]: good catch!

Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx]: I realized the mistake in a later PR and fixed it, but might as well fix it here. Thanks for pointing this out.

File: capstone-frontend/src/components/ClubPage.jsx [hunk 3]
```diff
@@ -81,10 +81,12 @@ class ClubPage extends Component {
       alert('Assignment creation failed. You do not own this club.');
       return;
     }
+    const dueDate = document.getElementById('due-date').value;
     let data = {
       'clubID': this.state.club.id,
       'text': e.target[0].value,
-      'whenCreated': (new Date()).toUTCString()
+      'whenCreated': (new Date()).toUTCString(),
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/ClubPage.jsx]: nit: You don't need the quotes on the left side of this

**Review by antoniolinhart — APPROVED** [review]
Looks good to me! Sorry for the late review

File: capstone-frontend/src/components/ClubPage.jsx (hunk 3 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx]: Will fix!

**Pull request merged**